src/corebluetooth.cs

```diff
@@ -43,6 +45,59 @@ interface CBCentralInitOptions {
 	interface CBManager {
 		[Export ("state", ArgumentSemantic.Assign)]
 		CBManagerState State { get; }
+
+		[iOS (13,0), TV (13,0), Watch (6,0), Mac (10, 15)]
```

Comment:
I think we still need the `only64: true` on the bindings...

Reply:
Actually if the build doesn't fail and tests don't fail, it shouldn't matter anymore (and any lingering need will be resolved before the final release anyway; it's on our todo list).

Reply:
I suppose since that we are not longer supporting 32b, it makes sense, nevertheless, I'll add it. It won't hurt